openedx/core/djangoapps/discussions/config/waffle.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module contains various configuration settings via
+waffle switches for the discussions app.
+"""
+
+from edx_toggles.toggles import LegacyWaffleFlagNamespace
+
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+WAFFLE_DISCUSSION_LEGACY_SETTINGS_NAMESPACE = LegacyWaffleFlagNamespace(name='discussions')
+
+# .. toggle_name: discussions.override_discussion_legacy_settings
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to override visibility settings for discussion settings in course advance settings.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-06-15
+# .. toggle_target_removal_date: 2021-12-31
+# .. toggle_warnings: Discussion settings and tab will be hidden on legacy experience
+# .. toggle_tickets: https://openedx.atlassian.net/browse/TNL-8389
+OVERRIDE_DISCUSSION_LEGACY_SETTINGS_FLAG = CourseWaffleFlag(
```

Comment:
This is mainly a cms related flag so should it be in cms directory just like the ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND?

Reply:
but we are using it in LMS as well... and also the `ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND` has to be used in LMS!

Reply:
@awaisdar001 as we discussed, should I move this to `discussions` app within `openedx` for a centralized place for the flag to exists?

Reply:
will create a followup PR for refactor and cater for this there!